cmd/mt-fakemetrics/cmd/datafeed.go

```diff
@@ -90,6 +91,8 @@ times %4d orgs: each %s, flushing %d metrics so rate of %d Hz. (%d total unique
 			// respecting where a previous flush left off, we need to start from
 			// the point after it.
 			var m int
+			ts = startingTs
```

Comment:
did you test this? this seems to reset the ts to the first one every time we flush.

Reply:
I didn't extensively test it, only for the case with a high `speedup`. I think you might be right though, in the case where the speedup is low that could cause a problem. I will need to think about this some more.

Reply:
I am working on a PR that will fix this differently